mods/weather/init.lua

```diff
@@ -79,7 +68,7 @@ end
 local os_time_0 = os.time()
 local t_offset = math.random(0, 300000)
 
-local function update_clouds()
+local function update_clouds(players)
```

Comment:
Why pass a list of players? ~~I'd just use the `playerset` upvalue.~~ Consider passing a set of players instead such that you may directly pass `playerset`. Note: This requires some further refactoring: You have to remove players from the set rather than setting them to `false`.